--- vendor/github.com/bwmarrin/discordgo/restapi.go	2017-12-05 00:21:53.000000000 +0100
+++ restapi.go	2017-12-05 00:21:00.000000000 +0100
@@ -38,7 +38,6 @@
 	ErrPruneDaysBounds         = errors.New("the number of days should be more than or equal to 1")
 	ErrGuildNoIcon             = errors.New("guild does not have an icon set")
 	ErrGuildNoSplash           = errors.New("guild does not have a splash set")
-	RequestsMade               int64
 )
 
 // Request is the same as RequestWithBucketID but the bucket id is the same as the urlStr
@@ -76,9 +75,6 @@
 		log.Printf("API REQUEST  PAYLOAD :: [%s]\n", string(b))
 	}
 
-	s.log(LogInformational, "discord api request %s %s\n", method, urlStr)
-	RequestsMade += 1
-
 	req, err := http.NewRequest(method, urlStr, bytes.NewBuffer(b))
 	if err != nil {
 		bucket.Release(nil)
