Index: restapi.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- restapi.go	(date 1528900458000)
+++ restapi.go	(date 1528334949000)
@@ -32,7 +32,6 @@

 // All error constants
 var (
-	RequestsMade               int64
 	ErrJSONUnmarshal           = errors.New("json unmarshal")
 	ErrStatusOffline           = errors.New("You can't set your Status to offline")
 	ErrVerificationLevelBounds = errors.New("VerificationLevel out of bounds, should be between 0 and 3")
@@ -77,9 +76,6 @@
 		log.Printf("API REQUEST  PAYLOAD :: [%s]\n", string(b))
 	}

-	s.log(LogInformational, "discord api request %s %s\n", method, urlStr)
-	RequestsMade += 1
-
 	req, err := http.NewRequest(method, urlStr, bytes.NewBuffer(b))
 	if err != nil {
 		bucket.Release(nil)
